### DISCUSSION TITLE
"Finding balance" - something went wrong

### DISCUSSION THREAD

**Discussion by wirnico** [iotaledger/firefly#7716] [Help & Support]
Hey there,
I hope to find someone who can help me here. I tried this morning to migrate from my old Trinity wallte to Firefly 1.2.0 with the explanation here: https://www.youtube.com/watch?v=lmRCwZeDtoc

Unfortunately I get stuck at the last step where in the IOTA Legacy app on my ledger the balance is searched. I always get the error by Firefly "something went wrong"

Does somebody have an idea how to continue?

Thanks, Nico

**Comment by begonaalvarezd** [comment]
Hi, Firefly 1.2.0 is deprecated, the network was recently upgraded so Firefly 1.x.y does not work anymore as it is not compatible with the new network upgrde. 
Regarding legacy migration: The current ability to auto-migrate tokens from the pre-Chrysalis IOTA network via the Firefly wallet have been stopped with the IOTA Stardust upgrade. Once IOTA smart contracts are available on the IOTA network, this feature will be restored and migrations can be processed automatically again
We will provide a migration tool to resume legacy migrations, but its not finished yet

**Comment by begonaalvarezd** [comment]
Will close the discussion as Firefly will not contain the legacy migration anymore but it will be a different tool 🙏🏼 
Please keep an open eye on the official communication, the IOTA Foundation will communicate it when the tool is ready

**Comment by wirnico** [comment]
Hi, thank you for the fast answer. Ok, good to know that it is not me. So I will wait for the migration tool? Do you know approximately when there will be an update out so I can check in again? Thx again for your good and fast support

**Reply by begonaalvarezd** [reply]
There is no ETA yet sorry 🙏🏼

**Reply by wirnico** [reply]
Hey @begonaalvarezd We were in contact back in November. Are there any news on migrating the IOTA? Is it working again? Thank you

**Reply by begonaalvarezd** [reply]
Yes, its out 🚀 https://blog.iota.org/iota-legacy-migration-tool/

**Reply by wirnico** [reply]
Thank you for your answer. I tried the process this morning but the tool always shows me 0 balance. What can cause this? I am using a ledger and bought the IOTA back in 2018 and were using Trinity before. I followed the instructions step by step. Thank you for any help

**Reply by begonaalvarezd** [reply]
Your funds might be in a different ledger index

**Reply by wirnico** [reply]
Unfortunately I have not written down the index number. But I could not imagine why I should have changed it from 0. Is there a way to find out the index number when using a ledger device?

**Reply by begonaalvarezd** [reply]
Some users had their funds in index 1,2.. you can try searching on couple low indexes like 1 or 2, i dont think there is a way to know in which index you have your funds if you dont know it

**Reply by wirnico** [reply]
It worked. I could export a Trinity log and I really changed the index. Found my coins. Thank you so much for the hint.

Just another noob question. I have now the screen message "Funds migrated"
I can now delete the IOTA Legacy app from my ledger, right?
Which is the new app compared to the previous Trinity where I can see my IOTA? Is it Firefly or any other?

**Reply by begonaalvarezd** [reply]
You can delete the IOTA Legacy Tool yes.
As for the wallet, you can use any trusted wallet now, eg Firefly, Bloom or TanglePay

**Reply by wirnico** [reply]
One last question. Someone on Discord told me this: "Hopefully you have carried this out, it’s a mandatory process 12 hours after migration to synchronize your stronghold and profile. Head over to the validator https://validatorsocket.com/
Select your wallet provider from the list e.g ( Ledger , Metamask etc ) 
Import your wallet provider and connect"

I am really careful to connect my wallet with anything. Is this safe? Is this really a thing I have to do after migration to the new network? Looks really spammy.

**Reply by kilianhln** [reply]
Scam! Do not trust people in Discord DMs.

**Reply by wirnico** [reply]
![Bildschirmfoto 2024-03-21 um 08 54 54](https://github.com/iotaledger/firefly/assets/12938801/1c7fd80d-259a-4c33-a7b1-d35e5948d0b4)

**Reply by wirnico** [reply]
What was irritating for me that he calls himself IOTA support.

**Reply by kilianhln** [reply]
Anyone can call themselves that. Its a new account, just look at the date.

**Comment by wirnico** [comment]
Just to sum up my problem. I migrated my IOTA to the new network 2 days ago. Is there anything left to do for me?
Do I have to do anything with stronghold? The coins are on my ledger and in my understanding they should be save there shouldn't they?

**Reply by begonaalvarezd** [reply]
If you migrated there is nothing left to do, keep your keys well saved and beware of scams.

**Reply by wirnico** [reply]
![Bildschirmfoto 2024-03-21 um 10 04 42](https://github.com/iotaledger/firefly/assets/12938801/b398c643-01ca-463e-bc0a-a3e9d1b59384)

By the way, I've been getting this error in the Firefly app for two days. Account creation was succesful but cannot login.
Bloom app works perfectly and my coins are showing up there.

**Reply by begonaalvarezd** [reply]
You can remove that profile and create it again if there is an issue with it, just import it again like you did in Bloom
